spec/e2e/proxy_spec.rb

```diff
@@ -133,26 +133,31 @@ def browser
   # end
 
   # TODO: examine "stop" else logic as it is skipping to "or http.http200(r)" choice each run
-  # it 'node correctly returns "bad redir error" when no redirect is given on stop' do
-  #   browser.goto "#{ctr_base_url}/nginx/stop"
-  #   expect(browser.selector(name: 'pre')).to eq('Error -- bad `redir` request ()')
-  # end
+  it '/nginx/stop needs a redirect' do
+    browser.goto "#{ctr_base_url}/nginx/stop"
+    expect(browser.selector(name: 'pre')).to eq('Error -- bad `redir` request ()')
+  end
 
-  it 'node correctly returns "bad redir error" when redirect to "github.com" given on stop' do
+  it '/nginx/stop wont redirect to another host' do
```

Comment:
I didn't really need to update these test names, but I just wanted something more succinct while I'm adding other tests.